vllm/outputs.py

```
@@ -122,6 +122,9 @@ def __init__(
         *,
         multi_modal_placeholders: Optional[MultiModalPlaceholderDict] = None,
         kv_transfer_params: Optional[dict[str, Any]] = None,
+        # Forward compatibility, code that uses args added in new release can
+        # still run with older versions of vLLM without breaking.
+        **kwargs: Any,
```

I think we should emit a deprecation warning instead of silently ignoring them, so the downstream libraries know about this problem

---

deprecation is about backward-compatbility, there we have forward-compatibility, so there's no arg to put warning on yet

---

How about just a general warning then? Stating that this argument is ignored

---

done